.github/workflows/website_tests.yml

```diff
@@ -0,0 +1,86 @@
+---
+name: Website Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DEBIAN_FRONTEND: noninteractive
+      LANG: en_US.UTF-8
+      LANGUAGE: en_US:en
+      LC_ALL: en_US.UTF-8
+      PODEXEC: 'podman exec --workdir /tmp/website --env-file .env ovirt-web-tests'
+    steps:
+      - name: "Checkout repo"
+        uses: actions/checkout@v2
+
+      - name: "Cache files"
+        uses: actions/cache@v2
+        id: website-cache
+        with:
+          path: tmp/.htmlproofer
+          key: htmlproofer
+
+      - name: "Check for nonbreaking space in markdowns"
+        run: |
+          if grep -P '\xa0' `find source/ -name "*.md"` ; then
+            echo '/!\ nonbreaking spaces have been found within the markdown (see previous log entries)'
```

Comment:
Not blocking, but the error message is a bit confusing.

Reply:
fixed

Reply:
not sure how to phrase this better. That's the name of the character.

Reply:
Oh, I mean more the part on previous logs entriess, since that's on another repo.